src/plugins/intel_gpu/src/plugin/ops/paged_attention.cpp

```diff
@@ -51,28 +51,35 @@ static void CreatePagedAttentionExtensionOp(ProgramBuilder& p, const std::shared
     prim.kv_heads_num = kv_heads_num;
     prim.heads_num = heads_num;
 
-    const size_t scale_idx = 9;
-    const size_t sliding_window_idx = 10;
-    const size_t alibi_idx = 11;
+    const size_t scale_idx = cldnn::paged_attention::PagedAttentionInputIdx::SCALE;
+    const size_t sliding_window_idx = cldnn::paged_attention::PagedAttentionInputIdx::SLIDING_WINDOW;
+    const size_t alibi_idx = cldnn::paged_attention::PagedAttentionInputIdx::ALIBI;
+    const size_t score_aggregation_idx = cldnn::paged_attention::PagedAttentionInputIdx::SCORE_AGGREGATION;
 
-    std::shared_ptr<ov::op::v0::Constant> scale_const = ov::as_type_ptr<ov::op::v0::Constant>(op->get_input_node_shared_ptr(scale_idx));
+    auto scale_const = ov::as_type_ptr<ov::op::v0::Constant>(op->get_input_node_shared_ptr(scale_idx));
     if (scale_const) {
         OPENVINO_ASSERT(ov::shape_size(scale_const->get_output_shape(0)) == 1);
         prim.scale_val = scale_const->cast_vector<float>()[0];
     } else {
         prim.scale_val = std::optional<float>();
     }
 
-    std::shared_ptr<ov::op::v0::Constant> sliding_windows_const = ov::as_type_ptr<ov::op::v0::Constant>(op->get_input_node_shared_ptr(sliding_window_idx));
+    auto sliding_windows_const = ov::as_type_ptr<ov::op::v0::Constant>(op->get_input_node_shared_ptr(sliding_window_idx));
     if (sliding_windows_const) {
         OPENVINO_ASSERT(ov::shape_size(sliding_windows_const->get_output_shape(0)) == 1);
         prim.sliding_window = sliding_windows_const->cast_vector<size_t>()[0];
     }
 
-    std::shared_ptr<ov::op::v0::Constant> alibi_const = ov::as_type_ptr<ov::op::v0::Constant>(op->get_input_node_shared_ptr(alibi_idx));
+    auto alibi_const = ov::as_type_ptr<ov::op::v0::Constant>(op->get_input_node_shared_ptr(alibi_idx));
     OPENVINO_ASSERT(alibi_const != nullptr);
     prim.has_alibi = ov::shape_size(alibi_const->get_output_shape(0)) > 0;
-    prim.has_rotated_blocks = op->get_input_size() == 16;
+
+    auto score_aggregation_input = ov::as_type_ptr<ov::op::v0::Parameter>(op->get_input_node_shared_ptr(score_aggregation_idx));
+    if (score_aggregation_input && score_aggregation_input->get_output_partial_shape(0).is_dynamic() && op->get_output_size() > 1) {
+        prim.has_score_aggregation = true;
+    }
+
+    prim.has_rotated_blocks = op->get_input_size() == 17;
```

Comment:
nit:  instead of number `17`, what about: `op->get_input_size() > ROTATION_TRIG_LUT`?